tools/service_image_uploader/downloader.toit

```diff
@@ -115,7 +117,7 @@ download config/cli.Config cache/cli.Cache ui/Ui parsed/cli.Parsed:
         cache-key := "snapshot-downloader/$name"
         snapshot := cache.get cache-key: | store/cli.FileStore |
           ui.info "Downloading $name."
-          store.save (client.storage.download --path="cli-snapshots/$name")
-
-        uuid := cache-snapshot snapshot --output-directory=output-directory
-        ui.info "Wrote CLI snapshot $uuid."
+          snapshot := client.storage.download --path="cli-snapshots/$name"
+          uuid := cache-snapshot snapshot --output-directory=output-directory
```

Comment:
Same here: the downloader now only stores the snapshot if the downloader hadn't cached anything for that file yet.

Reply:
Probably worth a short comment in the code.

Reply:
done.